crates/agentgateway/src/llm/anthropic.rs

```diff
@@ -1076,7 +1117,14 @@ pub(super) mod types {
 
 	#[derive(Clone, Serialize, Deserialize, Debug, Eq, PartialEq)]
 	pub struct MessageDeltaUsage {
```

Comment:
I'm not confident all 4 of these fields are already set? https://github.com/anthropics/anthropic-sdk-typescript/blob/4131d6ff28043ba675d7a8cdd673c25da5ebd4eb/src/resources/messages/messages.ts#L502

Do we need to wrap them in Option<>?